src/bridge/EspressoTEEVerifier.sol

```diff
@@ -25,20 +25,18 @@ import {IEspressoTEEVerifier} from "./IEspressoTEEVerifier.sol";
  *         from automata to verify the quote. Along with some additional verification logic.
  */
 contract EspressoTEEVerifier is IEspressoTEEVerifier, Ownable2Step {
-    event MREnclaveSet(bytes32 indexed mrEnclave);
-    event MRSignerSet(bytes32 indexed mrSigner);
-
     using BytesUtils for bytes;
 
     // V3QuoteVerififer contract from automata to verify the quote
     V3QuoteVerifier public quoteVerifier;
-    bytes32 public mrEnclave;
-    bytes32 public mrSigner;
+
+    mapping(bytes32 => bool) public mrEnclaves;
+    mapping(bytes32 => bool) public mrSigners;
```

Comment:
Is it ok that these are allowed to vary independently? Or is there a relationship between MR_ENCLAVE and MR_SIGNER?

For example say we have (enclave, signer) = (e1, s1) and then we do an upgrade to (e2, s2). Someone could then verify a batch using (e1, s2) or (e2, s1). Is that ok?

Reply:
I thought it was okay? I think even (e1, s1) or (e2, s2) is also okay. Do you see a reason why we should not support this?

Reply:
I have no idea haha you're the enclave expert. Just stood out to me as a potential subtlety. But if it's ok then I think this is all good